Dockerfile

```diff
@@ -50,16 +50,18 @@ RUN apt-get update \
   && cd $SENAITE_INSTANCE_HOME/src/senaite.indexer && git checkout main \
   && git clone  https://github.com/mekomsolutions/senaite.monkeypatches.git $SENAITE_INSTANCE_HOME/src/senaite.monkeypatches\ 
   && cd $SENAITE_INSTANCE_HOME/src/senaite.monkeypatches && git checkout main
-# Buildout
 
+# Buildout
 RUN cd $SENAITE_INSTANCE_HOME \
   && pip install -r requirements.txt \
   && buildout -c develop.cfg \
   && ln -s $SENAITE_FILESTORAGE/ var/filestorage \
   && ln -s $SENAITE_BLOBSTORAGE/ var/blobstorage \
   && chown -R senaite:senaite $SENAITE_HOME $SENAITE_DATA \
   # Cleanup
-  && apt-get purge -y --auto-remove $(grep -vE "^\s*#" /build_deps.txt  | tr "\n" " ") 
+  && apt-get purge -y --auto-remove $(grep -vE "^\s*#" /build_deps.txt  | tr "\n" " ") \
+  && rm -rf /$SENAITE_HOME/buildout-cache \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
This is taken from [pro Dockerfile](https://github.com/mekomsolutions/senaite-docker-ozonepro/pull/10/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L58-L61)